### PR TITLE
Provide usage docs

### LINKED ISSUE #27: Documenting usage
Provide documentation on using this tool in the readme.

Documentation should cover:
- changelog fragment creation
- changelog building
- configuration options

### PR DESCRIPTION
**Pull request by endorama**
Add usage docs and concepts.

Closes #27

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Adding a changelog fragment
+
+To create a new fragment:
+
+```
+$ elastic-agent-changelog-tool new a-changeset-filename
+```
+
+A new file will be created in the changelog fragments folder (default to `changelog/fragmennts`).
```
Comment (ph) [inline docs/usage.md:26]:
```suggestion
A new file will be created in the changelog fragments folder (default to `changelog/fragments).
```

**Review by ph — APPROVED** [review]
Changes LGTM, 

@endorama There is one thing missing from this PR, I see we mentioned what command but it would be nice to see the results of the command:

- What does a fragment look like?
- What a consolided changelog look likes?


Consider describing a user journey, from a PR to the resulting consolidated changelog.

**Comment by endorama** [comment]
@ph thanks for the suggestion! I'll update this PR.

File: docs/concepts.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+A **changelog entry** is an entry in the changelog, representing a documented changeset.
+
+Changelog entries are structured and enriched to provide all information needed to build the consolidated changelog.
+
+## Consolidated changelog
+
+The **consolidated changelog** is a list of changelog entries.
+
+Is created from a list of changelog fragments, enriched and structured. It can be manually edited where needed.
```
[the remark this answers is not in the document]
Comment (endorama) [reply docs/concepts.md:35]: Fragments file are prefixed with UNIX timestamp. When listing and reading the files from filesystem the timestamp force a creation time based sorting.

I [added](https://github.com/elastic/elastic-agent-changelog-tool/pull/29/files#diff-dbcdc80a52bc07ddc07abd2461ddb889265712596d5d9a095bd4a299e29dd87d) the changelog for this tool 0.1.0 version to this PR, you can see items are sorted from oldest (top) to newest (bottom).

**Review by ph — APPROVED** [review]

**Pull request merged**